source/MRViewer/MRObjectTransformWidget.h

```diff
@@ -57,15 +57,19 @@ class MRVIEWER_CLASS ObjectTransformWidget : public MultiListener<MouseDownListe
     enum AxisTransformMode
     {
         // object moves along an axis
-        Translation,
+        AxisTranslation,
         // object inflates or deflates along an axis depending on drag direction (away from center or toward center respectively)
-        Scaling,
+        AxisScaling,
```

Comment:
can we have three options here instead of `uniformScaling_` flag?

Reply:
Implemented.